Makefile

```
@@ -156,6 +156,7 @@ test-pipeline-rider-e2e:
 test-monovertex-rider-e2e:
 test-rollback-e2e:
 test-force-drain-e2e:
+test-force-drain-backward-compatibility-e2e:
```

checkmake (0.3.2)

[warning] Target "test-force-drain-backward-compatibility-e2e" should be declared PHONY.

(phonydeclared)